src/content/rohlik.tsx

```diff
@@ -3,10 +3,12 @@ import { GlobalStateProvider } from "../components/store/StoreProvider.tsx";
 import { render } from "./render";
 import { HashRouter as Router } from "react-router-dom";
 
-render(
-  <GlobalStateProvider>
-    <Router>
-      <Rohlik />
-    </Router>
-  </GlobalStateProvider>,
-);
+setTimeout(() => {
```

Comment:
ten set timeout se mi zda ze je u mne dulezitej, kor kdyz mam otevrenou konzoli, zda se mi, ze kdyz ten nas plugin naskoci driv, nez jejich javascript, zacne se to prat skrz jejich reconciliatio, tak to takhle obchazim, navrhuju to zachovat.